src/main/qss.js

```diff
@@ -606,35 +618,62 @@ gpii.app.qssWrapper.populateLanguageSettingOptions = function (settingOptions, l
 
 gpii.app.qssWrapper.loadSettings = function (assetsManager, installedLanguages, locale, messageBundles, settingOptions, settingsFixturePath, siteConfig) {
     var availableSettings = fluid.require(settingsFixturePath), // list of all available buttons
-        loadedSettings = availableSettings; // by default we are getting all of the buttons
+        loadedSettings = availableSettings, // by default we are getting all of the buttons
+        multipleir = 1000;
```

Comment:
typo multipleir -> multiplier. Please add comment that this value is only supplied in order to avoid rounding errors when comparing values with the bounds

Reply:
Please also add a comment explaining that this multiplier should correspond with the precision chosen in qssBaseStepperWidget